hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -193,7 +193,13 @@ int SaraNcpClient::init(const NcpClientConfig& conf) {
     waitReadyRetries_ = 0;
     registrationTimeout_ = REGISTRATION_TIMEOUT;
     resetRegistrationState();
-    ncpPowerState(modemPowerState() ? NcpPowerState::ON : NcpPowerState::OFF);
+    if (modemPowerState()) {
```

Comment:
@XuGuohui What about Quectel-based devices?

Reply:
yep, I'm going to test the Quectel-based platforms.